zappa/zappa.py

```diff
@@ -310,8 +313,13 @@ def create_lambda_zip(self, prefix='lambda_package', handler_file=None,
         print("Packaging project as zip...")
 
         if not venv:
+            conda_mode = False
             if 'VIRTUAL_ENV' in os.environ:
                 venv = os.environ['VIRTUAL_ENV']
+            elif 'CONDA_ENV_PATH' in os.environ:
```

Comment:
may want to check both CONDA_ENV_PATH and CONDA_PREFIX while people have various versions of conda installed.